webpack.config.js

```diff
@@ -26,8 +26,13 @@ const webpackConfig = {
 	module: {
 		rules: [
 			{
-				test: /\.(t|j)sx?$/,
-				use: [ 'ts-loader', 'babel-loader' ],
+				test: /\.tsx?$/,
+				use: [ 'babel-loader', 'ts-loader' ],
```

Comment:
From the discussion above I understand we decided to keep both loaders for now, but is the order correct here? I see it's been switched :)

Reply:
I checked on my Typescript contents from Deposits Information PR and it worked well. Because it runs on TypeScript file and not on the compiled output I assume order is of little significance here.